backend/src/main/resources/db/migration/V1.32.0__NOTIF-299_notifications-db-cleaner.sql

```diff
@@ -0,0 +1,25 @@
+-- This stored procedure deletes the event log entries that are no longer needed.
+-- It is executed from an OpenShift CronJob.
+CREATE PROCEDURE cleanEventLog() AS $$
+DECLARE
+    deleted INTEGER;
+BEGIN
+    RAISE INFO '% Event log purge starting. Entries older than 31 days will be deleted.', NOW();
+    DELETE FROM notification_history WHERE created < now() - interval '31 days';
```

Comment:
nitpick: looks like interval is a reserved keyword, should it be in upper case? (same below and with `now`)
```suggestion
    DELETE FROM notification_history WHERE created < NOW() - INTERVAL '31 days';
```